crates/sui-source-validation-service/src/lib.rs

```diff
@@ -72,13 +79,38 @@ pub struct SourceInfo {
     pub source: Option<String>,
 }
 
+#[derive(Eq, PartialEq, Clone, Deserialize, Debug, Ord, PartialOrd)]
+pub enum Network {
```

Comment:
The fact that this is case sensitive and upper-cased when serialized and lowercased when displayed is a potential source of confusion.  With `serde` you can rename enum variants so that `Network::Mainnet` is represented by `mainnet` everywhere, see, e.g.:

```suggestion
#[serde(rename_all = "lowercase")]
pub enum Network {
```